### ISSUE TITLE
[Feature] Triplet Marginal Loss

### ISSUE THREAD

**Issue by cvnad1** [google-deepmind/optax#1118]
I would like to add the Triplet Marginal Loss Function.

Reference: https://pytorch.org/docs/stable/_modules/torch/nn/functional.html#triplet_margin_loss

**Comment by cvnad1** [comment]
@vroulet Hey sorry for a little delay

I completed the code for the loss function and am currently adding tests. Once the tests are added, I will create a PR.

Thanks for giving me the location where to add !!!

**Comment by vroulet** [comment]
Done in #1120
